doc/Command-Reference.md

```diff
@@ -6564,6 +6568,301 @@ This command is used to set the counter polling interval. Default is 20 seconds.
 
 Go Back To [Beginning of the document](#) or [Beginning of this section](#sflow)
 
+## SNMP
+
+### SNMP Show commands
+
+**show runningconfiguration snmp**
+
+This command displays the global SNMP configuration that includes the location, contact, community, and user setttings.
+
+- Usage:
+  ```
+  show runningconfiguration snmp
+  ```
+
+- Example:
+  ```
+  admin@switch1:~$ show runningconfiguration snmp 
```

Comment:
Suggest changing prompt to `admin@sonic:~$` to align with rest of document. Please do throughout.